examples/web/src/components/MonacoEditor.vue

```diff
@@ -18,7 +18,7 @@ const props = withDefaults(
 // const isDark = useDarkMode(
 const model = defineModel<string>()
 
-const monacoEditorRef = ref<HTMLElement | null>(null)
+const monacoEditorRef: Ref<HTMLElement | null> = ref(null)
```

Comment:
I’ve seen that kind of change a few times, is that different/better for some reason? Just wondering.

Reply:
There is a Vue TS bug that throws if you do it using the generic. Very annoying.